src/heartbeatClient.ts

```diff
@@ -3,7 +3,7 @@ import { Logger } from '@map-colonies/js-logger';
 import { httpClientConfig } from './models/utils';
 
 export class HeartbeatClient extends HttpClient {
-  private readonly intervalDictionary: Record<string, NodeJS.Timer | undefined>;
+  private readonly intervalDictionary: Record<string, NodeJS.Timeout | undefined>;
```

Comment:
Shouldn't it remain **NodeJS.Timer** as we use the "setInterval" method?

Reply:
Then please remove 
<img width="645" height="530" alt="image" src="https://github.com/user-attachments/assets/30cec8f9-c4ab-4181-80e8-ff602f96d363" />

Reply:
done